lib/modules/modules.go

```diff
@@ -213,6 +231,16 @@ const (
 	ProductTypeEUB ProductType = 2
 )
 
+// SupportType is the type of support.
+type SupportType int32
```

Comment:
Maybe use the proto enum directly, instead of duping it here?

Reply:
Agreed, thank you.